tests/sources/test_google_drive.py

```diff
@@ -1474,11 +1455,24 @@ async def test_prepare_access_control_doc(user, groups, access_control_doc):
             Aiogoogle, "as_service_account", return_value=expected_response_object
         ):
             with mock.patch.object(ServiceAccountManager, "refresh"):
-                assert (
-                    access_control_doc
-                    == await source.prepare_single_access_control_document(user=user)
+                result = await source.prepare_single_access_control_document(user=user)
+
+                # mustache templates are hard to verify, so let's strip the mustache
+                filtered_input_source = " ".join(
+                    access_control_doc["query"]["template"]["source"]
+                    .replace("{{toJson}}access_control{{/toJson}}", "[]")
```

Comment:
Does the `#` in the toJson template string need to be included in this replace as well?

Something like `.replace("{{#toJson}}access_control{{/toJson}}", "[]")`

Reply:
Huh. Yes, the `#` should matter. This must not be doing what I thought it was.

Turns out I can remove all the complexity I added to these tests. 🙈